apps/perf-test-react-components/src/scenarios/ColorPicker.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { ColorPicker, ColorArea, ColorSlider, AlphaSlider } from '@fluentui/react-color-picker-preview';
```

Comment:
please remove these from this application (follow up) - its not being used and will be removed soon

Reply:
done